### ISSUE TITLE
webp support has been part of Qt 5 since 5.3

### ISSUE THREAD

**Issue by liangqi** [spillerrec/qt5-webp-plugin#1]
http://doc.qt.io/qt-5/qtimageformats-index.html
https://github.com/qtproject/qtimageformats/commit/4522b350e53471c2ebc6d4692736ee4708445b66

If it missed any features which are needed, you can help Qt Project to improve it.

**Comment by spillerrec** [comment]
I wrote this back before 5.3 was released (2 months before that commit apparently). Quickly looking through the code I can see that it does not support animations, though I never added that myself. I don't even think is was a part of WebP back then either.
Is it back-ported to Qt 4 btw?

**Comment by liangqi** [comment]
Feb 13, 2014 is just the committed time, the change started from May 14, 2013, https://codereview.qt-project.org/#/c/56026/ 

Nobody plans to back-ported it to Qt 4.

**Comment by spillerrec** [comment]
Indeed it is. It is strange that I didn't find it when I searched for it back then. (I did find a image viewer with added support by building it into the application, and a plugin which didn't support encoding though.)

There is still a lot of Qt 4 applications around, most noticeable KDE. And the image plugins actually adds functionality to existing applications.
I wanted to get thumbnails for WebP in Dolphin (KDE's file manager), and all I had to do to get it working with Qt 4 was two `#ifdef`s: https://github.com/spillerrec/qt-webp-plugin/commit/bac129f8280693f97c249d89282b21d042380903

Also check out: https://github.com/ArchangelSDY/qwebpa
It is a fork of the Qt version, adding animation support. Seems like a simple change that could easily be added, though I just gave it a quick look and there is no diff on github to easily tell if it is the case.

Anyway, I'm not quite sure what your intend is. Informing me about the development of Qt? Promoting contributing to Qt? Investigating short-comings of current Qt? I don't want to close this issue before I know that its purpose has been fulfilled.
